src/rtop_init.ml

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env ocaml
 #require "menhirLib";;
+#require "re.str";;
```

Comment:
Gah! I didn't know this file existed. I kept just making the other change and not being able to fix it. Thank you :)